vhd/packer/configure-windows-vhd.ps1

```diff
@@ -188,7 +188,7 @@ function Install-WindowsPatches {
 
     # KB4558998 contains August 11, 2020 cumulative updates for Windows Server 2019
     # https://www.catalog.update.microsoft.com/Search.aspx?q=KB4565349

```

Comment:
Maybe we may just change this comment into a template instead for each specific patch.

Reply:
Will update

Reply:
Updated comment, thanks!